chpl/chpl-service/src/main/java/gov/healthit/chpl/attestation/manager/AttestationExceptionEmail.java

```
@@ -0,0 +1,84 @@
+        }
+
+        public void send(AttestationPeriodDeveloperException attestationException) throws EmailNotSentException {
+            List<UserDTO> recipients = resourcePermissions.getAllUsersOnDeveloper(
+                    Developer.builder().id(attestationException.getDeveloper().getId()).build());
+
+            chplEmailFactory.emailBuilder()
+                .recipients(recipients.stream()
+                        .map(user -> user.getEmail())
+                        .collect(Collectors.toList()))
+                .subject(emailSubject)
+                .htmlMessage(createExceptionHtmlMessage(attestationException))
+                .sendEmail();
+
+        }
+
+        private String createExceptionHtmlMessage(AttestationPeriodDeveloperException attestationException) {
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMM dd, YYYY");
+            return chplHtmlEmailBuilder.initialize()
+                    .heading("Developer Attestations Submission Reopened")
+                    .paragraph("", String.format(emailBody, getActingBody(), formatter.format(attestationException.getExceptionEnd())))
+                    .footer(true)
+                    .build();
+        }
+
+        private String getActingBody() {
+            if (resourcePermissions.isUserRoleAcbAdmin()) {
+                List<CertificationBodyDTO> acbsForUser = resourcePermissions.getAllAcbsForCurrentUser();
+                if (CollectionUtils.isEmpty(acbsForUser)) {
+                    LOGGER.warn("No ACBs were found for the current user " + AuthUtil.getCurrentUser().getSubjectName());
+                    return "an ONC-ACB";
+                }
+                return acbsForUser.get(0).getName();
+            } else if (resourcePermissions.isUserRoleAdmin()) {
+                return "CHPL Admin";
+            } else if (resourcePermissions.isUserRoleOnc()) {
+                return "ONC";
+            } else {
+                return "";
```

We probably need some other default here, or should throw an exception. There's a default for the ACB; can we use something similar? Maybe "an administrator" and LOG a warning?